src/Orc.FileAssociation/Services/FileAssociationService.cs

```diff
@@ -102,5 +117,26 @@ public async Task UndoAssociationFilesWithApplicationAsync(ApplicationInfo appli
             }
 
         }
+
+        public virtual async Task OpenPropertiesWindowForExtensionAsync(string extension)
+        {
+            var appPath = AppDomain.CurrentDomain.BaseDirectory;
+            var resourcesPath = Path.Combine(appPath, "Resources\\");
```

Comment:
Please only use Path.Combine (it uses the environment folder separator char) , remove the \\